dgea/dgea_scvi.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import pandas as pd
+import scanpy as sc
+import random
+import anndata as ad
+import os
+
+import torch
+from scvi.model import SCANVI, SCVI
+
+def scanvi_dgea(adata:ad.AnnData, groupby:str, reference:str, alternative:str):
+     
+    directory_model = "model_scvi_test/"
```

Comment:
This is not how we configure things here

Reply:
yes I know